canonical-web-and-design/maas-ui
# Display a favicon

## Done
- Add a favicon to the single-spa app.

## QA
- Run `dotrun`.
- Load a react and angular page, both should have a favicon.

## Fixes
Fixes https://github.com/canonical-web-and-design/maas-squad/issues/2003.
